spec/factories/events/review_comments.rb

```diff
@@ -23,9 +23,11 @@
 #
 
 FactoryBot.define do
+  sequence(:review_comment_id, 100)
+
   factory :review_comment, class: Events::ReviewComment do
-    github_id { Faker::Number.unique.number(digits: 4) }
-    body { Faker::Quote.singular_siegler }
+    github_id { generate(:review_comment_id) }
+    body { generate(:body) }
```

Comment:
Here you can use the sequence as an implicit attribute, like so

```suggestion
    body
```

Same for other occurrences in this PR.